src/js/components/Form/stories/ArrayOfFormFields.js

```diff
@@ -55,10 +55,18 @@ export const ArrayOfFormFields = () => {
               return undefined;
             },
           ]}
-        />
+        >
+          <TextInput
+            aria-required
+            aria-label="phone number"
```

Comment:
I don't think this needs `aria-label` since it has a `<label for="phone number">` (`htmlFor` in our case.)